### PR TITLE
First batch of doc cleanups

### PR DESCRIPTION
**Pull request by rocky**
There is still a lot of work to do. And this is going to take a bit of time. 

A suggestion is to commit this first batch and then iterate. 

A problem with large PRs that hang out for a while is that they require lots of merges or rebases to keep them in synch.

@jansorg Your thoughts?

### CODE REVIEW

File: command/next.sh [hunk 1]
```diff
@@ -24,14 +24,19 @@
...
 
-Single step an statement skipping functions. This is sometimes called
+Step one statement ignoring steps into function calls at this level. This is sometimes called
 'step over' or 'step through'.
 
-If *count* is given, stepping occurs that many times before
-stopping. Otherwise *count* is one. *count* an be an arithmetic
-expression.
+With an integer argument, perform ``next`` that many times. However if
+an exception occurs at this level, or we *return*, *yield* or the
```
Comment (jansorg) [inline command/next.sh]: I don't think there's `yield` or threads in Bash.

File: docs/commands/breakpoints/condition.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
+.. index:: condition
+.. _condition:
+
+Add Condition to a Breakpoint (condition)
+-----------------------------------------
+
+**condition** *bp_number* *condition*
+
+*bp_number* is a breakpoint number. *condition* is a zsh expression
```
Comment (jansorg) [inline docs/commands/breakpoints/condition.rst]: `zsh expression` -> `Bash expression`

File: docs/commands/breakpoints/tbreak.rst [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+**tbreak** [ *location* ] [**if** *condition*]
+
+With a line number argument, set a break there in the current file.
+With a function name, set a break at first executable line of that
+function.  Without argument, set a breakpoint at current location.  If
+a second argument is `if`, subequent arguments given an expression
+which must evaluate to true before the breakpoint is honored.
+
+The location line number may be prefixed with a filename or module
+name and a colon. Files is searched for using *sys.path*, and the `.py`
```
Comment (jansorg) [inline docs/commands/breakpoints/tbreak.rst]: I don't think `*.py` is correct in this context.

File: docs/commands/breakpoints/tbreak.rst [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+Examples:
++++++++++
+
+::
+
+   tbreak     # Break where we are current stopped at
+   tbreak 10  # Break on line 10 of the file we are currently stopped at
+   tbreak os.path.join # Break in function os.path.join
+   tbreak os.path:45   # Break on line 45 of os.path
+   tbreak myfile.py:45 # Break on line 45 of myfile.py
```
Comment (jansorg) [inline docs/commands/breakpoints/tbreak.rst]: These examples seem to be for Python, not Bash.

File: docs/commands/data/display.rst [hunk 5]
```diff
@@ -0,0 +1,24 @@
+.. index:: display
+.. _display:
+
+Set a Display Expression (`display`)
+------------------------------------
+
+**display** [ *format* ] *expression*
```
Comment (jansorg) [inline docs/commands/data/display.rst]: Is bashdb really supporting `format`? According to `help display` it's not supporting it.

File: docs/commands/data/eval.rst [hunk 6]
```diff
@@ -0,0 +1,52 @@
+.. index:: eval
+.. _eval:
+
+Evaluate a Zsh Expression (`eval`)
```
Comment (jansorg) [inline docs/commands/data/eval.rst]: `Zsh` -> `Bash`

File: docs/commands/data/load.rst [hunk 7]
```diff
@@ -0,0 +1,13 @@
+.. index:: load
+.. _load:
+
+Read in a Zsh Script (`load`)
+-----------------------------
+
+**load** *zsh-script*
+
+Read in lines of a *zsh-script*.
```
Comment (jansorg) [inline docs/commands/data/load.rst]: `zsh-script` -> `bash-script`

File: docs/commands/data/load.rst [hunk 8]
```diff
@@ -0,0 +1,13 @@
+.. index:: load
+.. _load:
+
+Read in a Zsh Script (`load`)
```
Comment (jansorg) [inline docs/commands/data/load.rst]: `Zsh` -> `Bash`

File: docs/commands/files/list.rst [hunk 9]
```diff
@@ -0,0 +1,54 @@
+.. index:: list
+.. _list:
+
+List Zsh Source Code (`list`)
```
Comment (jansorg) [inline docs/commands/files/list.rst:4]: `Zsh` -> `Bash`

File: docs/commands/info/breakpoints.rst [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+* The \"Disp\" column contains one of \"keep\", \"del\", the disposition of the breakpoint after it gets hit.
+* The \"enb\" column indicates whether the breakpoint is enabled.
+* The \"Where\" column indicates where the breakpoint is located.
+
+Example:
+++++++++
+
+::
+
+   zshdb<4> info breakpoints
```
Comment (jansorg) [inline docs/commands/info/breakpoints.rst]: `zshdb` -> `bashdb`

File: docs/commands/info/source.rst [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+**info source**
+
+Information about the current zsh script file.
+
+Example:
+++++++++
+
+::
+
+    zshdb<1> info source
```
Comment (jansorg) [inline docs/commands/info/source.rst]: `zshdb` -> `bashdb`

File: docs/commands/info/variables.rst [hunk 12]
```diff
@@ -0,0 +1,31 @@
+.. index:: info; variables
+.. _info_variables:
+
+Info Variables
+----------------
+
+**info variables** [*property*]
```
Comment (jansorg) [inline docs/commands/info/variables.rst]: I think that `-p` is missing here.

File: docs/commands/running/debug.rst [hunk 13]
```diff
@@ -0,0 +1,17 @@
+.. index:: debug
+.. _debug:
+
+Recursive Debugging (`debug`)
+-----------------------------
+
+**debug** [*zsh-script* [*args*...]]
```
Comment (jansorg) [inline docs/commands/running/debug.rst]: `zsh-script` -> `bash-script`

File: docs/commands/running/debug.rst [hunk 14]
```diff
@@ -0,0 +1,17 @@
...
+
+Recursively debug into *zsh-script*.
```
Comment (jansorg) [inline docs/commands/running/debug.rst]: `zsh-script` -> `bash-script`

File: docs/commands/set/autoeval.rst [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
+
+Auto-Evaluation of Unrecognized Debugger Commands (`set auto eval`)
+-------------------------------------------------------------------
+
+**set autoeval** [ **on** | **off** ]
+
+Evaluate unrecognized debugger commands.
+
+Often inside the debugger, one would like to be able to run arbitrary
+zsh commands without having to preface expressions with
```
Comment (jansorg) [inline docs/commands/set/autoeval.rst]: `zsh` -> `bash`

File: docs/commands/set/autolist.rst [hunk 16]
```diff
@@ -0,0 +1,47 @@
...
+**set autolist** [ **on** | **off** ]
+
+Run the :ref:`list <list>` command every time you stop in the
+debugger.
+
+With this, you will get output like:
+
+::
+
+    -> 1 from subprocess import Popen, PIPE
```
Comment (jansorg) [inline docs/commands/set/autolist.rst]: This is Python code.

File: docs/commands/set/debug.rst [hunk 17]
```diff
@@ -0,0 +1,17 @@
+.. index:: debug
+.. _set_debug:
+
+Recursive Debugging (`debug`)
+-----------------------------
+
+**debug** [*zsh-script* [*args*...]]
```
Comment (jansorg) [inline docs/commands/set/debug.rst]: `zsh-script` -> `bash-script`

File: docs/commands/set/debug.rst [hunk 18]
```diff
@@ -0,0 +1,17 @@
...
+
+Recursively debug into *zsh-script*.
```
Comment (jansorg) [inline docs/commands/set/debug.rst]: `zsh-script` -> `bash-script`

**Review by jansorg — CHANGES_REQUESTED** [review]
For now, I've read half of the files. I'll continue later with the review.
The docs seem to come from zshdb. A global search/replace for `zsh`->`Bash` and `zshdb`->`bashdb` would probably remove a bunch of the things highlighted in the review.

**Comment by jansorg** [comment]
Iterating on the docs seems best to me. Reviewing a huge PR is usually hard, too.

**Comment by rocky** [comment]
These changes and more are in. I haven't been able to get readthedocs to build because this is in a branch that it is currently not recognizing.

**Comment by rocky** [comment]
Ok - docs are now created at https://bashdb.readthedocs.io/en/latest/

Feel free to add links to BashdbPro, but let's merge this first.

**Review by jansorg — APPROVED** [review]
Thanks! 
I glanced over the .rst files, but didn't check the .png/.gif files or the build setup.

**Comment by rocky** [comment]
Thanks! BTW - I haven't forgotten about the other PR's. Busy with things right now. Also, I sent email regarding possibilities for releases.

**Pull request merged**